x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/_legend.scss

```diff
@@ -0,0 +1,38 @@
+.mlJobMapLegend__container {
+  background-color: '#FFFFFF';
+}
+
+.mlJobMapLegend__indexPattern {
+  height: $euiSizeM;
+  width: $euiSizeM;
+  background-color: '#FFFFFF';
+  border: 1px solid $euiColorVis2;
+  transform: rotate(45deg);
+  display: 'inline-block';
+}
+
+.mlJobMapLegend__transform {
+  height: $euiSizeM;
+  width: $euiSizeM;
+  background-color: '#FFFFFF';
+  border: 1px solid $euiColorVis1;
+  display: 'inline-block';
+}
+
+.mlJobMapLegend__analytics {
+  height: $euiSizeM;
+  width: $euiSizeM;
+  background-color: '#FFFFFF';
+  border: 1px solid $euiColorVis0;
+  border-radius: 50%;
+  display: 'inline-block';
+}
+
+.mlJobMapLegend__inferenceModel {
```

Comment:
Might need renaming to `trainedModel` if you switch 'inference model' node type to 'trained model' - see above.